app/request/views.py

```diff
@@ -180,9 +181,10 @@ def view(request_id):
 
     :return: redirect to view request page
     """
-    current_request = Requests.query.filter_by(id=request_id).one()
-
-    if not current_request.agency.is_active:
+    try:
+        current_request = Requests.query.filter_by(id=request_id).one()
+        assert current_request.agency.is_active
+    except (NoResultFound, AssertionError):
```

Comment:
I would suggest splitting these up so that we can do appropriate error logging for this:
```python
try:
    current_request = Requests.query.filter_by(id=request_id).one()
    assert current_request.agency.is_active
except NoResultFound:
     log("SOME MESSAGE")
     return abort(404)
except AssertionError:
     log("Some Other Message")
     return abort(404)
```